client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/ILMDocumentationIT.java

```diff
@@ -119,6 +123,92 @@ public void onFailure(Exception e) {
 
     }
 
+    public void testGetLifecyclePolicy() throws IOException {
+        RestHighLevelClient client = highLevelClient();
+
+        // Set up some policies so we have something to get
+        {
+            Map<String, Phase> phases = new HashMap<>();
+            Map<String, LifecycleAction> hotActions = new HashMap<>();
+            hotActions.put(RolloverAction.NAME, new RolloverAction(
+                new ByteSizeValue(50, ByteSizeUnit.GB), null, null));
+            phases.put("hot", new Phase("hot", TimeValue.ZERO, hotActions));
+
+            Map<String, LifecycleAction> deleteActions =
+                Collections.singletonMap(DeleteAction.NAME,
+                    new DeleteAction());
+            phases.put("delete",
+                new Phase("delete",
+                    new TimeValue(90, TimeUnit.DAYS), deleteActions));
+
+            LifecyclePolicy policy = new LifecyclePolicy("my_policy",
+                phases);
+            PutLifecyclePolicyRequest putRequest =
+                new PutLifecyclePolicyRequest(policy);
+            LifecyclePolicy policy2 = new LifecyclePolicy("other_policy",
+                phases);
```

Comment:
Could we make the two policies slightly different (even if its only the min age on the phase so we can differentiate them when we assert the policy that is returned later?